src/main/java/pizza/AnswersToInquiries.java

```diff
@@ -0,0 +1,59 @@
+package pizza;
+
+import java.util.List;
+
+
+public class AnswersToInquiries {
```

Comment:
metody super, ale zobacz, ten kod można użyć tylko z tymi pizzami które stworzyłeś w main, z niczym innym. To nie jest dobre rozwiązanie jeśli chcemy żeby nasz kod był reużywalny. Zamiast tego może w każdej z tych metod przyjmuj parametr List<Pizza> pizzas i na nich dokonuj tych różnych operacji ;)